router/server.go

```diff
@@ -191,6 +201,13 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		http.Error(w, "", http.StatusTeapot)
 		return
 	}
+
+	// This redirects all HTTP request to use HTTPS for all connections which are not local to the server.
+	if (s.configHttp.RedirectHTTP && !isSecure && req.Proto == "HTTP/1.1" && req.Host != "127.0.0.1" && req.Host != "localhost") {
+		http.Redirect(w, req, "https://" + req.Host + req.URL.String(), http.StatusMovedPermanently)
```

Comment:
I think I recall that the thinking was that if the hostname is any IP address, then don't redirect. This will future-proof us for when we're not just running an instance on a single server. In that case, I imagine our services will use an IP address for the router.